spec/output/generator_spec_elixir/union.x/int_union.ex

```diff
@@ -30,18 +30,20 @@ defmodule MyXDR.IntUnion do
   defstruct [:value, :type]
 
   @spec new(value :: value(), type :: Int.t()) :: t()
-  def new(value, %Int{} = type), do: %__MODULE__{value: value, type: type}
+  def new(value, type), do: %__MODULE__{value: value, type: type}
```

Comment:
The spec needs to be updated. Now `type` is not an `Int` struct

![image](https://github.com/kommitters/xdrgen/assets/39246879/3e60a14f-1a65-4e89-856d-845d464c336a)